staging/src/k8s.io/apiserver/pkg/util/flag/flags.go

```diff
@@ -47,6 +48,7 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+	pflag.CommandLine.SetOutput(os.Stdout)
```

Comment:
in what cases is this written to? will this mix errors or warnings into output?

Reply:
It only effects help information,the output produced in running process is another thing

Reply:
`SetOutput` sets the destination for usage and error messages.
I don't think it is safe to specify `os.Stdout` here.

Reply:
@dixudx I have realized my mistake,I am working on if we have any solution here.

Reply:
Would modifying your command to `kubelet --help 2>&1 | grep default` fix your use case?

Reply:
yeah... it's not uncomment to send help text to stderr... grep itself does this

```
$ grep --help | grep context
usage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]

$ grep --help 2>&1 | grep context
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
```

Reply:
OK,Thanks!